sdk/storage/azure-storage-blob/azure/storage/blob/_shared/models.py

```diff
@@ -323,11 +325,13 @@ class AccountSasPermissions(object):
     :param bool process:
         Valid for the following Object resource type only: queue messages.
     """
-    def __init__(self, read=False, write=False, delete=False, list=False,  # pylint: disable=redefined-builtin
+    def __init__(self, read=False, write=False, delete=False, delete_version=False,
```

Comment:
I think this class is exposed publicly right?
The new parameter will need to go at the end of the method signature - otherwise it will be a breaking change for anyone who specified arguments positionally.